vm/src/obj/objdict.rs

```diff
@@ -315,11 +318,8 @@ impl Iterator for DictIter {
     type Item = (PyObjectRef, PyObjectRef);
 
     fn next(&mut self) -> Option<Self::Item> {
-        match self.dict.entries.borrow().next_entry(self.position) {
-            Some((new_position, key, value)) => {
-                self.position = new_position;
-                Some((key.clone(), value.clone()))
-            }
```

Comment:
You could just do `.cloned()` here.

Reply:
I don't think it works through the tuple.